docs/_data/nakadi-event-bus-api.yaml

```diff
@@ -2858,6 +2861,29 @@ definitions:
         description: |
           Number of milliseconds that Nakadi stores events published to this event type.
 
+
+  EventAuthField:
+    type: object
+    description: |
+      Parameters to define per-event authorization. This field can be used with an authorization plugin
+      to differentiate which authorized consumers for an event type can receive an event published to
+      the event type. In case the field is null or not present in the event, it will be allowed to be read
+      by all authorized consumers.
+      Once defined, the definition for EventAuthField for an event-type cannot be updated or removed.
+    properties:
+      path:
+        type: string
+        description: |
+          Path to a string field in the event that contains the a classifier, which can be used by an
+          authorization plugin (together with definitions in the authorization section) to define who can read the
+          event. The field may not be present in the schema definition for the event type. In case the field is not
+          present, it will be available for reading for all authorized consumers.
+      type:
+        type: string
+        description: |
+          Indicator of the type of the data (classifier) the field contains (eg: `retailer_id` or `application_name`).
```

Comment:
You can also use `example` field from `open-api`.

Reply:
It might also be worth explaining that its plugins responsibility to handle these values and this is installation specific.

Reply:
I don't really get meaninng of this field - it seems that it's just for informational purposes...